ui/src/app/services/swagger-fetch-clients.ts

```diff
@@ -181,6 +183,8 @@ export function bindApiClients(conf: FetchConfiguration) {
     registerApiClient(
       ctor,
       new (class extends ctor {
+        basePath: string;
```

Comment:
Anonymous class needed properties declared or the constructor would show an error.